services/util/test_doubles/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-add_library(services.services_test_doubles EXCLUDE_FROM_ALL STATIC)
+add_library(services.services_test_doubles ${EMIL_EXCLUDE_FROM_ALL} STATIC)
```

This should have been an INTERFACE library since it has no exported symbols.

```suggestion
add_library(services.services_test_doubles ${EMIL_EXCLUDE_FROM_ALL} INTERFACE)
```